Mathlib/GroupTheory/MonoidLocalization/Basic.lean

```diff
@@ -1330,3 +1330,56 @@ protected def localizationMap : S.LocalizationMap R[S⁻¹] := Localization.mono
 protected noncomputable def equivMonoidLocalization : Localization S ≃* R[S⁻¹] := MulEquiv.refl _
 
 end OreLocalization
+
```

Comment:
GIven that this file is already long, and this seem like a nice split point can we move this to a new file? That also gives a place to link the wikipedia page and disambiguate the categorical meaning of "GrothendieckGroup".